code/modules/telesci/gps.dm

```diff
@@ -123,8 +123,8 @@ var/list/SPS_list = list()
 			to_chat(usr, "<span class = 'caution'>The GPS needs to be kept in your active hand!</span>")
 			return TRUE
 		a = strict_ascii(a)
-		if(length(a) < 4 || length(a) > 5)
-			to_chat(usr, "<span class = 'caution'>The tag must be between four and five characters long!</span>")
+		if(length(a) > 5)
```

Comment:
are you sure you didn't forget to set the min to 1?

Reply:
what a beautiful var name `a` is. Pretty sure an empty input is being handled by ```if(!a) return TRUE``` a few lines above

Reply:
hmm wait, if `strict_ascii(a)` removes something then you can end up with an empty variable

Reply:
ill look into it